backend/api/tests/test_AuthorProfile.py

```diff
@@ -13,39 +13,110 @@ class AuthorProfileTestCase(TestCase):
-        )
-        self.author_id = get_author_id(self.authorProfile.host, str(self.authorProfile.id), False)
+        self.user2 = User.objects.create_user(username=self.username2, password=self.password2)
+        self.user3 = User.objects.create_user(username=self.username3, password=self.password3)
+        self.user4 = User.objects.create_user(username=self.username4, password=self.password4)
+
+        self.authorProfile = AuthorProfile.objects.create(host="http://127.0.0.1:5454/",
+                                                          displayName="Lara Croft",
+                                                          github="http://github.com/laracroft",
+                                                          user=self.user)
+
+        self.authorProfile2 = AuthorProfile.objects.create(host="http://127.0.0.1:5454/",
+                                                           displayName="Lara Croft number 2",
+                                                           github="http://github.com/laracroft2",
+                                                           user=self.user2)
+
+        self.authorProfile3 = AuthorProfile.objects.create(host="http://127.0.0.1:5454/",
+                                                           displayName="Lara Croft number 3",
+                                                           github="http://github.com/laracroft3",
+                                                           user=self.user3)
+
+        self.authorProfile4 = AuthorProfile.objects.create(host="http://127.0.0.1:5454/",
+                                                           displayName="Lara Croft number 4",
+                                                           github="http://github.com/laracroft4",
+                                                           user=self.user4)
+
+        self.user_id = get_author_id(self.authorProfile.host, self.authorProfile.id, False)
+        self.user_id2 = get_author_id(self.authorProfile2.host, self.authorProfile2.id, False)
+        self.user_id3 = get_author_id(self.authorProfile3.host, self.authorProfile3.id, False)
+        self.user_id4 = get_author_id(self.authorProfile4.host, self.authorProfile4.id, False)
+
+        self.user_id_escaped = get_author_id(self.authorProfile.host, self.authorProfile.id, True)
+
+        Follow.objects.create(authorA=self.user_id,
+                              authorB=self.user_id2,
+                              status="FRIENDS")
+
+        Follow.objects.create(authorA=self.user_id,
+                              authorB=self.user_id3,
+                              status="FOLLOWING")
+
+        Follow.objects.create(authorA=self.user_id,
+                              authorB=self.user_id4,
+                              status="FRIENDS")
+
+        self.expected_output = {
+            'id': 'http://127.0.0.1:5454/author/{}'.format(self.authorProfile.id), 
+            'host': 'http://127.0.0.1:5454/', 
```

Comment:
change host, display name etc to reference the value from the authorprofile object instead of hard coding it for easier future changes